### PR TITLE
Ws 931 dependency parser polished

### PR DESCRIPTION
**Pull request by dgoldfarble**

### CODE REVIEW

File: json/src/main/java/com/basistech/rosette/apimodel/jackson/DependencyDataMixin.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.basistech.rosette.apimodel.jackson;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class DependencyDataMixin extends BaseMixin {
```
Comment (guruxu) [inline json/src/main/java/com/basistech/rosette/apimodel/jackson/DependencyDataMixin.java]: `DependencyDataMixin` -> `DependencyMixin`?

File: model/src/main/java/com/basistech/rosette/apimodel/DependencyParserResponse.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.basistech.rosette.apimodel;
+
+import java.util.List;
+import java.util.Objects;
+
+public class DependencyParserResponse extends Response {
```
Comment (guruxu) [inline model/src/main/java/com/basistech/rosette/apimodel/DependencyParserResponse.java]: `DependencyParserResponse` -> `DependenciesResponse`

File: json/src/main/java/com/basistech/rosette/apimodel/jackson/DependencyParserResponseMixin.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+package com.basistech.rosette.apimodel.jackson;
+
+import com.basistech.rosette.apimodel.Dependency;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class DependencyParserResponseMixin extends BaseMixin {
```
Comment (guruxu) [inline json/src/main/java/com/basistech/rosette/apimodel/jackson/DependencyParserResponseMixin.java]: `DependenecyParserResponseMixin` -> `DependenciesResponseMixin`

**Comment by dgoldfarble** [comment]
All set.

**Pull request merged**